From 5efdac319bea6c000b17af77b7d1b3ba1dce6432 Mon Sep 17 00:00:00 2001
From: Namhyung Kim <namhyung@gmail.com>
Date: Sat, 21 Jan 2017 16:38:23 +0900
Subject: [PATCH] bash_completion: journalctl: Complete -t option values

The -t or --identifier requires a syslog identifier.

(cherry picked from commit 28b6b5de3f6c8943e83afe4e213cdb6a8c7e1a72)
---
 shell-completion/bash/journalctl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/shell-completion/bash/journalctl b/shell-completion/bash/journalctl
index ded92cb73a..c90a114497 100644
--- a/shell-completion/bash/journalctl
+++ b/shell-completion/bash/journalctl
@@ -82,6 +82,9 @@ _journalctl() {
                         --user-unit)
                                 comps=$(journalctl -F '_SYSTEMD_USER_UNIT' 2>/dev/null)
                         ;;
+                        --identifier|-t)
+                                comps=$(journalctl -F 'SYSLOG_IDENTIFIER' 2>/dev/null)
+                        ;;
                         *)
                                 return 0
                         ;;
